### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by buijs-dev**

### CODE REVIEW

File: gradlew
Comment (Xyndra) [inline gradlew:1]: I think this is supposed to be in the gitignore

Comment (buijs-dev) [reply gradlew:1]: It's better to just add it so you can build the code right after pulling it.

File: lib/kore/src/main/kotlin/dev/buijs/klutter/kore/project/Environment.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package dev.buijs.klutter.kore.project
+
+enum class OperatingSystem(val value: String) {
+    WINDOWS("windows"),
+    MACOS("macos")
```
Comment (Xyndra) [inline lib/kore/src/main/kotlin/dev/buijs/klutter/kore/project/Environment.kt:5]: don't forget linux

Comment (buijs-dev) [reply lib/kore/src/main/kotlin/dev/buijs/klutter/kore/project/Environment.kt:5]: Good point, thanks! Need to test on Linux to.

**Pull request merged**